### DISCUSSION TITLE
Option to rename domains for the /sites overview page

### DISCUSSION THREAD

**Discussion by metmarkosaric** [plausible/analytics#1029] [Anything Else]
"I'm tracking projects in subfolders but due to the URL length, I can not easily see the full URL on the /sites page so I can not
easily see whether it's the German or the English version (for example). Maybe you could integrate that I could define a custom display name to use an abbreviation/short name so I can improve the overview page."

(feedback via email)

**Comment by mtarnovan** [comment]
That would be very helpful for us too.

**Comment by manukall** [comment]
Hi @metmarkosaric,

 any plans for implementing something like this soon? 

We're using plausible to provide our publishers at [Steady](https://steadyhq.com) with traffic stats about their sites. The publishers can change the URL of their site. After doing that, they should still be able to see old stats, though. This is actually pretty easy to do with Plausible by just sending data from the new site to the existing plausible domain, but they would still see the old name in the embed.

**Reply by metmarkosaric** [reply]
Hi @manukall! Yeah, it can work like that with the only downside being the domain name in the top left being the old one until we build the functionality to edit the domain. We plan to allow this but there are no updates and no short term plans that I can tell you about. We are a small team with limited development resources so need to focus on most frequently requested features one at a time. Current focus is on the data import and performance improvements. Thanks for understanding!

**Reply by manukall** [reply]
I completely understand. Sounds like us. :-)

Is adding this functionality a bigger undertaking? Do you think it's something we could do ourselves? Would you be open to a PR?

**Reply by metmarkosaric** [reply]
From my understanding, this feature requires some big changes in the database. The domain names are unique identifiers in our database, so we'll have to change this to a random numeric site-ID instead. Then the change in the UI to allow people to rename the site will be simple.

**Reply by manukall** [reply]
Oh, that sounds like a pretty big change indeed.

Would adding an explicit "display name" make sense, instead? This would be used as title in graphs, but wouldn't have any function besides. Sounds like it would address the original feature request, too.

**Reply by ukutaht** [reply]
It would work as a band-aid but I think it would make the eventual transition even more difficult. Sorry, I know this is a frustrating limitation but we really do plan to fix it!

**Reply by emzo** [reply]
When you do transition to using  numeric site-IDs, please also give the option of adding a display name as well (independent of the domain/ID) to make it easier to find the site you're looking for at a glance when you have a large number of sites.

**Reply by GarveScott-Lodge** [reply]
Alternatively, as has been requested in a number of other threads on the forum, the ability to move sites into subfolders or label them in some way would be very useful and may work as a workaround in this case.

**Comment by manukall** [comment]
Alright, got it. Thanks for explaining. :-)

**Comment by JeFawk** [comment]
> "I'm tracking projects in subfolders but due to the URL length, I can not easily see the full URL on the /sites page so I can not easily see whether it's the German or the English version (for example). Maybe you could integrate that I could define a custom display name to use an abbreviation/short name so I can improve the overview page."
> 
> (feedback via email)

This would be super useful for us too since our tracking goes deep 2 subfolders

![image](https://user-images.githubusercontent.com/7268697/183834299-f07ee792-f276-4587-97ea-2bc38fa0697d.png)
